src/main/java/com/WebController/LoginController.java

```diff
@@ -0,0 +1,99 @@
+package com.WebController;
+
+import com.Model.ConfirmationToken;
+import com.Model.User;
+import com.Repository.ConfirmationTokenRepository;
+import com.Repository.UserRepository;
+import com.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
```

Comment:
Be careful you're not adding any behaviour here that clients who are not going through the web interface wouldn't be able to access.